packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -653,18 +656,35 @@ class BackdropFilter extends SingleChildRenderObjectWidget {
   /// widgets.
   const BackdropFilter.grouped({
     super.key,
-    required this.filter,
+    this.filter,
```

Comment:
same, consider deprecation and reimplementing with ImageFilterConfig.filter

Reply:
(Discussing this issue in https://github.com/flutter/flutter/pull/175473#discussion_r2377012519)